vendor/github.com/konsorten/go-windows-terminal-sequences/sequences.go

```diff
@@ -27,7 +26,7 @@ func EnableVirtualTerminalProcessing(stream syscall.Handle, enable bool) error {
 		mode &^= ENABLE_VIRTUAL_TERMINAL_PROCESSING
 	}
 
-	ret, _, err := setConsoleMode.Call(uintptr(unsafe.Pointer(stream)), uintptr(mode))
```

Comment:
Ah. looks like this fixes the fail using go 1.14 on windows

Reply:
Yes, I think so; I spotted it in my notifications that logrus updated it, so thought to open a PR here (kept it separate from the logrus update in case we wanted to cherry-pick changes individually, although I don't think we'll be updating (release) branches to Go 1.14 directly